### ISSUE TITLE
Allow a "no-image" mode for mosviz

### ISSUE THREAD

**Issue by eteq** [spacetelescope/jdaviz#489]
Right now mosviz requires an on-sky image.  But as outlined in https://github.com/spacetelescope/jdaviz/blob/main/notebooks/concepts/Mosviz%20concept%20notebook.ipynb (rendering correctly here: https://nbviewer.jupyter.org/github/spacetelescope/jdaviz/blob/main/notebooks/concepts/Mosviz%20concept%20notebook.ipynb), there's very much a uscase for "mosviz without the images".  This issue is to allow something like what's in that notebook in the "Load other MOS data" section.

**Comment by rosteen** [comment]
#502 added the ability to at least load a dataset into Mosviz that does not include images, leaving the image viewer unpopulated. Doing some sort of dynamic layout where the image viewer only appears if images are loaded would be much more work, and was left as a future improvement for now.

**Comment by eteq** [comment]
Gotcha, thanks @rosteen - I think we should leave this open then, because as the concept notebook highlighted originally the desire *is* for the "no image section shows up at all". But agreed that if that's a bigger technical challenge it should be a follow-on PR!

**Comment by pllim** [comment]
Duplicate of #363

**Comment by havok2063** [comment]
This issue was created to describe leftover discussion and work to (partially) solve #363.  This was left open as a reminder that more work was needed to remove the image viewer from Mosviz when no images were input.   I'll reopen so this work does not get lost, unless there's another specific issue tracking it.

**Comment by pllim** [comment]
Oh, okay. Sounds awfully similar to  #363 .

**Comment by havok2063** [comment]
Yeah it is.  This was broken up into 
1. bug fix to immediately address #363 but leave the empty viewer and 
2.  implement the more desired thing of removing the image viewer when no images are input.

**Comment by pllim** [comment]
I don't understand... The Mosviz example notebook has an example that loads without images and it works for me. Is this really still an issue, or can we close this and #363 ?

**Comment by havok2063** [comment]
You can load Mosviz without images but it currently still displays an empty image viewer.  Ideally that image viewer would be removed.  It's a question of whether the current implementation is complete enough.  I think we can safely close #363 but we should either leave this open to track item 2 above, or we can close this but open a new ticket specifically regarding item 2.

**Comment by eteq** [comment]
Yeah, I agree with @havok2063 here - my intent in that concept notebook was exactly this, that if you don't provide an image, there's *no viewer*, and similar if you don't provide a 2d spectrum.
